cmd/migration/v0.9.1-v1/mapper/trade.go

```diff
@@ -0,0 +1,84 @@
+package mapper
+
+import (
+	v091domain "github.com/tdex-network/tdex-daemon/cmd/migration/v0.9.1-v1/v091-domain"
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+)
+
+func (m *mapperService) FromV091TradesToV1Trades(
+	trades []*v091domain.Trade,
+) ([]*domain.Trade, error) {
+	res := make([]*domain.Trade, 0, len(trades))
+	for _, v := range trades {
+		trade, err := m.fromV091TradeToV1Trade(v)
+		if err != nil {
+			return nil, err
+		}
+		res = append(res, trade)
+	}
+
+	return res, nil
+}
+
+func (m *mapperService) fromV091TradeToV1Trade(
+	trade *v091domain.Trade,
+) (*domain.Trade, error) {
+	market, err := m.v091RepoManager.MarketRepository().GetMarketByAssets(
+		trade.MarketBaseAsset, trade.MarketQuoteAsset,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return &domain.Trade{
+		Id:               trade.ID.String(),
+		Type:             0, // TODO
```

Comment:
this is the trade type, either BUY or SELL

Reply:
look from trader standpoint: if a market with ABC as base asset and XYZ as quote asset, when the user is receiveing the ABC (ie. base asset) it's a trade of type BUY, vice versa if its receiving the quote asset is a SELL

Reply:
oh sorry i was strongly convinced the v0 domain stored the trade type but it's really not!
What tiero suggested is ok but I want to add that the v0 domain has a concept of fixed base/quote asset in conifg that means all markets must have the same base/quote asset. Since this info is not stored in the datadir, we may want to add this param as a flag (we should mimic the behavior of the daemon's v0 fixed base asset as default value).

Reply:
Done